collector/filesystem_common.go

```diff
@@ -126,30 +126,37 @@ func (c *filesystemCollector) Update(ch chan<- prometheus.Metric) (err error) {
 	if err != nil {
 		return err
 	}
+	// Make sure we expose a metric once, even if there are multiple mounts
+	seen := map[filesystemStats]bool{}
```

Comment:
Oh good catch.. Right, value could be different.. While that should be the case, I came up with a better way. What do you think about this?